src/plugins/intel_gpu/src/plugin/transformations_pipeline.cpp

```diff
@@ -235,7 +236,11 @@ void TransformationsPipeline::apply(std::shared_ptr<ov::Model> func) {
 
         auto pass_config = manager.get_pass_config();
         pass_config->disable<ov::pass::EyeDecomposition>();
-        pass_config->enable<ov::pass::ConvertCompressedOnlyToLegacy>();
+
+        // disable conversion to legacy and use the new mixed precision
+        // in which precision sensitive nodes are kept in FP32
+        pass_config->disable<ov::pass::ConvertCompressedOnlyToLegacy>();
+        manager.register_pass<ov::pass::ConvertCompressedToMixedPrecision>();
```

Comment:
Are we sure that `ConvertCompressedToMixedPrecision` is triggered in the expected order in the pipeline using this approach? May be we need to register this transformation in disabled mode in CommonOptimization and enable it here?

Reply:
Thanks! That's a good point. I didn't think of that. I will double check if placement is correct

Reply:
Moved to the beginning `CommonOptimizations` because when we call optimization passes we should've get rid of all decompression Converts. So now `ConvertCompressedToMixedPrecision` is called instead of `ConvertCompressedOnlyToLegacy` at the exactly same order